<launch>
    <node name="line_filter" pkg="cyber_is_filters" type="line_filter" output="screen">
        <param name="line_detector_topic" value="/line_detector"/>
        <param name="max_limit" value="false"/>
        <param name="threshold0" value="3800"/>
        <param name="threshold1" value="3800"/>
        <param name="threshold2" value="3800"/>
        <param name="threshold3" value="3800"/>
        <param name="threshold4" value="3800"/>

    </node>
    <node name="magnet_filter" pkg="cyber_is_filters" type="magnet_filter" output="screen">
        <param name="magnet_detector_topic" value="/magnet"/>
        <param name="max_limit" value="false"/>
        <param name="min_value" value="5000"/>
        <param name="max_value" value="3500"/>
    </node>
</launch>
